.github/workflows/publish-hotfix-release.yml

```diff
@@ -0,0 +1,267 @@
+#################################################################################
+#  Copyright (c) 2021,2023 Contributors to the Eclipse Foundation
+#
+#  See the NOTICE file(s) distributed with this work for additional
+#  information regarding copyright ownership.
+#
+#  This program and the accompanying materials are made available under the
+#  terms of the Apache License, Version 2.0 which is available at
+#  https://www.apache.org/licenses/LICENSE-2.0.
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations
+#  under the License.
+#
+#  SPDX-License-Identifier: Apache-2.0
+#################################################################################
+
+
+---
+name: "Publish hotfix release"
+
+# Temporary workflow to enable the release of bugfix versions.
+# Should be reworked as this file contains duplicate code from draft-new-release and publish-new-release
+# This workflow, as is, should be executed after the hotfix branch is created and all needed PRs are merged
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'The hotfix version you want to release.'
+        required: true
+
+jobs:
+  # Gate: Check release version presence
+  validate-release:
+    name: Validate if hotfix can be released
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref_name, 'hotfix/')
+    steps:
+      - uses: mukunku/tag-exists-action@v1.6.0
+        id: check-tag
+        with:
+          tag: ${{ inputs.version }}
+
+      - if: steps.check-tag.outputs.exists == 'true'
+        name: Exit on tag existance
+        run: |
+          echo "Tag already exists! Please choose another tag."
+          exit 1
+
+  release-version:
+    name: Determine release version
```

Comment:
why is this job needed? it doesn't do anything other than setting the `RELEASE_VERSION` variable. instead, you could just use `${{ inputs.version }}` everywhere and add `validate-release` to the `needs` dependencies

Reply:
Addressed in [c240841](https://github.com/eclipse-tractusx/tractusx-edc/pull/1511/commits/c24084146ae9dad6a139561ae477c8eef49b2764)